packages/canner-layouts/src/Body.js

```diff
@@ -1,60 +1,41 @@
 // @flow
 
-import  React from 'react';
+import React, { useContext } from 'react';
 // $FlowFixMe: antd Breadcrumb
 import {Breadcrumb, Icon} from 'antd';
-import {Item} from 'canner-helpers';
+import {Context, Item} from 'canner-helpers';
+import {FORM_TYPE} from 'canner/src/hooks/useFormType';
+
+import {DefaultCreateBody, DefaultListBody, DefaultUpdateBody} from './defaultBody';
 
 type Props = {
   id: string,
   title: string,
   description: string,
 
   schema: Object,
-  routes: Array<string>
+  routes: Array<string>,
+
+  createComponent?: any,
```

Comment:
Use [react component type](https://flow.org/en/docs/react/types/),
```suggestion
  createComponent?: React$Component<Props, State>,
  updateComponent?: React$Component<Props, State>,
  listComponent?: React$Component<Props, State>,
```

Reply:
My question is check flow type cannot pass if use `React$Component<Props, State>`.

Thank you for your other suggestions. commit pushed.